src/devices/GrovePi/Sensors/ISensor.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Iot.Device.GrovePiDevice.Models;
+using System.Collections.Generic;
+
+namespace Iot.Device.GrovePiDevice.Sensors
+{
+    /// <summary>
+	/// Interface for a sensor 
+	/// </summary>
+    public interface ISensor<T>
```

Comment:
if this can be internal please change (can't remember if this was in any public APIs or just base classes)

Reply:
If people want to build their own class with this interface, I would leave it public.